cogs/playerstats.py

```diff
@@ -41,11 +56,15 @@ def __init__(self, bot):
         help="Get highest goalscorers for a specific team, defaults to Arsenal.")
     async def goals(self, ctx, team: str = 'Arsenal'):
         if team.lower() not in self.CLUB_ID_TRANSLATIONS:
-            return await ctx.send(f'Sorry, I couldn\'t find a team with the name {team},'
+            return await ctx.send(f'Sorry, I couldn\'t find a team with the name {team},\n'
+                                  f'If your team has a space in the middle, wrap the team in quotes(ie: "west ham")\n'
```

Comment:
Is there any way we could do this for them? Seems less than customer friendly to require users to use non-normal  entries.

Reply:
I have resolved this. stay tuned.